File: 2023/java/src/test/java/org/frieder/aoc/day8/TestDay8B.java
package org.frieder.aoc.day8;

import org.frieder.aoc.day8.b.Solution8B;
import org.junit.Test;

import java.io.IOException;

import static org.junit.Assert.assertEquals;

public class TestDay8B {
    @Test
    public void testExample() throws IOException {
        assertEquals(6D, Solution8B.getResult("../resources/8/test_b.txt"), 0);
    }

    @Test
    public void testResult() throws IOException {
        assertEquals(14299763833181D, Solution8B.getResult("../resources/8/input.txt"), 0);
    }
}

File: 2023/php/test/TestDay8B.php
<?php declare(strict_types=1);

$loader = require __DIR__ . '/../vendor/autoload.php';

use day8\b\Solution8B;
use PHPUnit\Framework\TestCase;
use utils\TestLogger;

final class TestDay8B extends TestCase
{

    public function testExample(): void
    {
        $this->assertSame(6, Solution8B::getResult("../resources/8/test_b.txt", TestLogger::createTestLogger()));
    }

    public function testResult(): void
    {
        $this->assertSame(14299763833181, Solution8B::getResult("../resources/8/input.txt", TestLogger::createTestLogger()));
    }

}